<ui version="4.0" >
 <class>BootLoaderWidget</class>
 <widget class="QWidget" name="BootLoaderWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLabel" name="textLabel1" >
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="text" >
      <string>&lt;b>Do you want to write boot loader (GRUB) to your hard disk's master boot record?&lt;/b>

&lt;p>You need a boot loader to boot your Pardus system.&lt;/p>

&lt;p>If you don't install a boot loader or choose to install to root partition, you will have to configure your other operating system's loader  (if you have one installed) to start 
your newly installed Pardus.&lt;/p></string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="margin" >
      <number>35</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>31</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Where do you want to install the bootlader ?</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="installFirstMBR" >
        <property name="text" >
         <string>Install to first bootable disk (recommended)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="installPart" >
        <property name="text" >
         <string>Install to Pardus installed partition</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="installMBR" >
        <property name="text" >
         <string>Install to selected disk belove</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="select_disk_label" >
          <property name="text" >
           <string>Select the disk you to install bootloader:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="device_list" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="noInstall" >
        <property name="text" >
         <string>Don't install bootloader</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>308</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>installFirstMBR</tabstop>
  <tabstop>installPart</tabstop>
  <tabstop>installMBR</tabstop>
  <tabstop>noInstall</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
